requirements/doc.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.9
```

Comment:
We should run `make upgrade` using `python 3.8` virtualenv since we haven't updated to `Python 3.9` yet.